snewpdag/plugins/True_VS_Fit.py

```diff
@@ -0,0 +1,79 @@
+"""
+Author: M. Colomer (marta.colomer@ulb.be)
+
+"""
+import logging
+import numpy as np
+
+from snewpdag.dag import Node
+
+class True_VS_Fit(Node):
+  def __init__(self, in_field, **kwargs):
+    self.map = {}
+
+    self.field = in_field
+
+    v = kwargs.pop('in_index', None)
+    self.index = tuple(v) if isinstance(v, list) else v
+    v = kwargs.pop('in_index2', None)
+    self.index2 = tuple(v) if isinstance(v, list) else v
```

Comment:
Would it be possible to change TimeDistDiff to puts its output in a simple payload field, instead of having all the index and index2 stuff?  It complicates downstream code, and isn't really necessary because all the history is already preserved in the history field.

Reply:
I fully agree, it would be convenient to modify TimeDistDiff and just output the 'tdelay' in a simple way with the number, instead of the complicated structure it outputs now.